### PR TITLE
#118 LCC metric has impediments

### PR DESCRIPTION
**Pull request by uurl**
For #118

**Issue**
LCC metric has impediments. Need to wait for resolving these impediments, add LCC to reports list and cover it by tests.

### CODE REVIEW

**Comment by uurl** [comment]
@rultor merge

**Comment by uurl** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` are banned at [#239](https://github.com/yegor256/jpeek/issues/239); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: src/main/java/org/jpeek/App.java [hunk 1]
```diff
@@ -68,12 +68,12 @@
  * @checkstyle MethodLengthCheck (500 lines)
  * @checkstyle JavaNCSSCheck (500 lines)
  *
- * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xml).
+ * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xsl).
```
Comment (yegor256) [inline src/main/java/org/jpeek/App.java]: I've seen this in 5 pull requests already. What's going on? Please, work in different branches. Here is the instruction: https://www.yegor256.com/2014/04/15/github-guidelines.html

**Comment by yegor256** [comment]
@uurl see above

**Pull request closed without merging**